package.json

```diff
@@ -71,7 +71,7 @@
     "@bazel/karma": "0.32.2",
     "@bazel/typescript": "0.32.2",
     "@firebase/app-types": "^0.3.2",
-    "@octokit/rest": "^15.9.4",
+    "@octokit/rest": "^16.28.7",
```

Comment:
you should add a script to package.json so we can do `yarn cherrypick` or something